add-on/src/lib/ipfs-proxy/acl-whitelist.json

```diff
@@ -0,0 +1,30 @@
+[
+  "block.get",
+  "block.stat",
+  "dag.get",
+  "dag.tree",
+  "dht.get",
+  "dht.findprovs",
+  "dht.findpeer",
+  "dht.query",
+  "files.cat",
+  "files.catPullStream",
+  "files.get",
+  "files.getReadableStream",
+  "files.getPullStream",
+  "files.ls",
+  "files.lsReadableStream",
+  "files.lsPullStream",
+  "object.get",
+  "object.data",
+  "object.links",
+  "object.stat",
+  "pin.ls",
+  "pubsub.subscribe",
+  "pubsub.unsubscribe",
+  "pubsub.peers",
+  "pubsub.ls",
+  "swarm.peers",
+  "swarm.addrs",
+  "swarm.localAddrs"
+]
```

Comment:
Too permissive?

Reply:
This is a whitelist right? Any other calls will by default be blocked? Think they are all fine, but I do worry about for example MFS being totally open for everyone with access to window.ipfs.

Reply:
I'd remove all `*.ls` from whitelist for now. Adding / reading specific hash can be done without explicit permission, but listing internal node state should be require confirmation.